docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -4,58 +4,59 @@ description: How to configure access and trust between two SSH and Kubernetes en
-located behind firewalls without any open TCP ports. The real-world usage examples of this
-capability include:
+The design of Trusted Clusters allows Teleport users to connect to compute
+infrastructure located behind firewalls without any open TCP ports. The
+real-world usage examples of this capability include:
 
 - Managed service providers (MSP) remotely managing the infrastructure of their clients.
-- Device manufacturers remotely maintaining computing appliances deployed on-premises.
-- Large cloud software vendors manage multiple data centers using a common proxy.
+- Device manufacturers remotely maintaining computing appliances deployed on premises.
+- Large cloud software vendors managing multiple data centers using a common proxy.
 
-**Example of a MSP provider using trusted cluster to obtain access to clients clusters.**
+Here is an example of an MSP using Trusted Clusters to obtain access to client clusters:
 ![MSP Example](../../../img/trusted-clusters/TrustedClusters-MSP.svg)
 
-The Trusted Clusters chapter in the Admin Guide
-offers an example of a simple configuration which:
+This guide will explain how to:
 
-- Uses a static cluster join token defined in a configuration file.
-- Does not cover inter-cluster Role-Based Access Control (RBAC).
-
-This guide's focus is on more in-depth coverage of trusted clusters features and will cover the following topics:
-
-- How to add and remove trusted clusters using CLI commands.
+- Add and remove Trusted Clusters using CLI commands.
 - Enable/disable trust between clusters.
 - Establish permissions mapping between clusters using Teleport roles.
 
-<Admonition
-  type="tip"
-  title="Teleport Node Tunneling"
->
-  If you have a large number of devices on different networks, such as managed IoT devices or a couple of nodes on a different network you can utilize [Teleport Node Tunneling](./adding-nodes.mdx).
-</Admonition>
+<Details title="Teleport Node Tunneling" scope={["enterprise", "oss"]} scopeOnly opened>
 
-## Introduction
+ If you have a large number of devices on different networks, such as managed
+ IoT devices, you can configure your Teleport Nodes to connect to your cluster
+ via Teleport Node Tunneling. Instead of connecting to the Auth Service
+ directly, a Node connects to the Proxy Service, and the Auth Service creates a
+ reverse tunnel to the Node.
 
-As explained in the [architecture document](../../architecture/overview.mdx#design-principles),
-Teleport can partition compute infrastructure into multiple clusters.
-A cluster is a group of SSH nodes connected to the cluster's *auth server*
-acting as a certificate authority (CA) for all users and nodes.
+ Learn more in [Adding Nodes to the Cluster](./adding-nodes.mdx).
+
+</Details>
+
+## How Trusted Clusters work
+
+Teleport can partition compute infrastructure into multiple clusters. A cluster
+is a group of Teleport SSH Nodes connected to the cluster's Auth Service, which
```

Comment:
```suggestion
is a group of Teleport resources connected to the cluster's Auth Service, which
```

Not sure of term (resources, agents, etc), but let's be clear that a cluster is more than SSH nodes and an Auth Service.